project_name/settings/base.py

```diff
@@ -95,10 +100,15 @@ def base_dir_join(*args):
 # Webpack
 WEBPACK_LOADER = {
     'DEFAULT': {
-        'CACHE': True,
+        'CACHE': False,
```

Comment:
CACHE should be actually False in local and True in production  
https://github.com/owais/django-webpack-loader#default-configuration